kvson_test.go

```diff
@@ -130,4 +130,8 @@ func TestExists(t *testing.T) {
 	if kvson.Exists("exists") == false {
 		t.Errorf("It should exist, but it doesn't")
 	}
+
+	if kvson.Delete("exists") == false {
```

Comment:
Golang anti pattern, instead of checking for true/false, Delete method should return error ( like all methods ).
This will allow in the future to use panic() or whatever the user wants.